# Sediment-Whole-Genome-Metagenome
A total of 6 samples from Kochi sediment of upper and lower part regions were analyzed using whole genome metagenome analysis.

## workflow and tools used
Quality control - Fastp
Assembly - MEGAHIT
Taxonomic classification - Kraken2
Custom analysis (Antibiotic resistance gene identification) - CARD, NCBI and ResFinder databases
Custom analysis (Heavy metal resistance gene identification) - NCBI
Statistical analysis (Top 20 genus and phylum level relative abundance) - Python (pandas, numpy and matplotlib)
Statistical analysis (Venn diagram analysis) & (Canonical correspondence analysis) - InteractiVenn & Python - pandas, sklearn.cross_decomposition, matplotlib and adjustText.
Diversity analysis (Alpha, Beta & Principal component analysis) - Python (Pandas, numpy and matplotlib).
Functional metagenomic analysis - KEGG, COG & SEED

## Data
The dataset consists of total a 6 samples from Kochi sediment upper and lower regions upper part (S1UP, S2UP, S3UP) and lower part (S1LP, S2LP, S3LP).

## Author 
Komala Devi S
